File: 01 Introdução/1044 Múltiplos/1044.java
import java.util.Scanner;

class Main {

  public static void main(String args[]) {
    Scanner sc = new Scanner(System.in);

    int A = sc.nextInt();
    int B = sc.nextInt();

    if (A % B == 0 || B % A == 0) {
      System.out.println("Sao Multiplos");
    } else {
      System.out.println("Nao sao Multiplos");
    }

    sc.close();
  }
}

// https://www.beecrowd.com.br/judge/pt/problems/view/1044
// Ranking 06156º, Time 0.095, Submiss Date 12/02/2023 23:52:54 (Java 14)

File: 01 Introdução/1044 Múltiplos/1044.py
num1, num2 = [int(x) for x in input().split(' ')]

if num1 % num2 == 0 or num2 % num1 == 0:
    print('Sao Multiplos')
else:
    print('Nao sao Multiplos')

# https://www.beecrowd.com.br/judge/pt/problems/view/1044
# Ranking 13101º, Time 0.049, Submiss Date 04/03/2023 01:37:47 (Python 3.9)
